packages/round-manager/src/features/round/FundContract.tsx

```diff
@@ -104,13 +101,16 @@ export default function FundContract(props: {
       matchingFundPayoutToken?.address ===
       "0x0000000000000000000000000000000000000000"
         ? undefined
-        : matchingFundPayoutToken?.address,
+        : (matchingFundPayoutToken?.address as `0x{string}`),
```

Comment:
For the future, is there any non-casting way of reading a string into a `0x{string}`? Couldn't find much about it